echo-core/src/main/java/com/netflix/spinnaker/echo/config/EchoRetrofitConfig.java

```diff
@@ -26,7 +25,7 @@
 @Slf4j
 public class EchoRetrofitConfig {
   @Bean
-  public Ok3Client ok3Client(OkHttp3ClientConfiguration okHttpClientConfig) {
-    return new Ok3Client(okHttpClientConfig.create().build());
+  public Ok3Client ok3Client() {
+    return new Ok3Client();
   }
```

Comment:
Why remove `OkHttp3ClientConfiguration`?

Reply:
because it was actually coming through with our custom SSL config which is not what we want anymore - that is all handled by the clientServiceProvider. @srekapalli and I tried to use his new `serviceClientProvider.getService` with `isSecure=false` but that was not working because as soon as `sslSocketFactory` even if it's the default factory the SSL communication was breaking (at least to slack). I will defer to @srekapalli to looking into that